Formula/notcurses.rb

```diff
@@ -30,11 +30,9 @@ def install
   end
 
   test do
-    # without the TERM definition, Notcurses is like a small child, lost in a
-    # horrible mall. of course, if the tests are run in some non-xterm
-    # environment, this choice might prove unfortunate.
-    # you have no chance to survive. make your time.
-    ENV["TERM"] = "xterm"
-    assert_match "notcurses", shell_output("#{bin}/notcurses-info")
+    # current homebrew CI runs with TERM=dumb. given that Notcurses explicitly
+    # does not support dumb terminals (i.e. those lacking the "cup" terminfo
+    # capability), we expect a failure here. all output will go to stderr.
+    assert_match "", shell_output("#{bin}/notcurses-info", 1)
```

Comment:
```suggestion
    assert_empty shell_output("#{bin}/notcurses-info", 1)
```

Reply:
Alternatively, you could redirect to stdout with `2>&1`, or similar.

Reply:
i applied the suggestion. i really don't want to make error strings part of any committed api.